letta/server/rest_api/interface.py

```diff
@@ -500,9 +496,6 @@ def _process_chunk_to_letta_style(
                     )
 
             elif self.inner_thoughts_in_kwargs and tool_call.function:
-                if self.use_assistant_message:
```

Comment:
why did we remove this?

Reply:
i don't think this is a field that's set anymore - it's always set to False

Reply:
since it is not passed in the API call